examples/pipelines/simple_pipeline/components/image_filtering/requirements.txt

```diff
@@ -0,0 +1 @@
+git+https://github.com/ml6team/express.git@b1855308ca9251da5ddd8e6b88c34bc1c082a71b#egg=express[datasets]
```

Comment:
I guess this will change to express[Polars] at some point. Are we still leveraging HF datasets?

Reply:
my bad I confused them 🙃 I meant Dask